e2e/e2e_test.go

```diff
@@ -601,3 +612,21 @@ openAPI:
 		})
 	}
 }
+
+func findSubCmd(cmd *cobra.Command, names ...string) *cobra.Command {
```

Comment:
you can probably can use https://godoc.org/github.com/spf13/cobra#Command.Find

Reply:
I looked for something like this, but for some reason I didn't find it. I have updated the PR to use the built-in cobra function.